### PR TITLE
Characters

### PR DESCRIPTION
**Pull request by Chulstro**
Completed form and creation functionality for characters. Needs some user catches for missing information.

### CODE REVIEW

File: app/controllers/characters_controller.rb [hunk 1]
```diff
@@ -2,4 +2,14 @@ class CharactersController < ApplicationController
...
+  def new
+    @new_character_info = CharacterFacade.create_form
+  end
+
+  def create
+    CharacterFacade.create_new(params["name"], params["lvl"], params["race"], params["class"])
+
+    redirect_to "/"
+  end
 end
```
Comment (Chulstro) [inline app/controllers/characters_controller.rb:15]: This facade wan't catch incomplete information. Need to add reroutes for missing info.

File: spec/features/characters/new_spec.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    fill_in 'name', with: "Chitter"
+    select '3', from: "lvl"
+    select 'Barbarian', from: 'class'
+    select 'Kenku', from: 'race'
+
+    click_on "Create Character"
+    expect(current_path).to eq("/")
+    expect(page).to have_content("Chitter")
+  end
+end
```
Comment (Chulstro) [inline spec/features/characters/new_spec.rb:26]: This could use tests for edge cases and not just basic functionality

**Comment by Chulstro** [comment]
Need to go into character model and make assertions for at least level and class
